src/test/java/org/opentripplanner/updater/trip/moduletests/delay/DelayedTest.java

```diff
@@ -72,11 +66,11 @@ void singleStopDelay() {
    */
   @Test
   void complexDelay() {
-    var env = RealtimeTestEnvironment.gtfs();
+    var env = RealtimeTestEnvironment.gtfs().withTrip2().build();
 
-    var tripId = env.trip2.getId().getId();
+    var tripId = TRIP_2_ID;
```

Comment:
We can probably get rid of this `tripId` variable now that it doesn't reallly improve readability anymore

Reply:
I reworked the handling of the IDs here: c97d90a2bff2948d4a62e253fc1fe9fb0b2a6dd6